wafer/schedule/templates/wafer.schedule/penta_schedule.xml

```diff
@@ -70,7 +78,8 @@
                         <description/>
                         <type/>
                     {% endif %}
-                    <conf_url>{{ items.item.get_url }}</conf_url>
+                    {# forcing https here is a bit horrible - make this configurable somewhere? #}
+                    <conf_url>https://{{ WAFER_CONFERENCE_DOMAIN }}{{ items.item.get_url }}</conf_url>
```

Comment:
I'm okay with leaving the conference URL relative if that helps (since those loading the pentabarf presumably know the URL its relative to).